app/src/org/commcare/views/media/MediaLayout.java

```diff
@@ -473,64 +337,75 @@ private View getInlineVideoView(String inlineVideoURI, RelativeLayout.LayoutPara
                     }
                     FirebaseAnalyticsUtil.reportInlineVideoPlayEvent(videoFilename, FileUtil.getDuration(videoFile), duration);
                 });
-
                 videoView.setOnClickListener(v -> ViewUtil.hideVirtualKeyboard((Activity)getContext()));
-
-                //These surprisingly get re-jiggered as soon as the video is loaded, so we
-                //just want to give it the _max_ bounds, it'll pick the limiter and shrink
-                //itself when it's ready.
-                viewLayoutParams.width = maxBounds[0];
-                viewLayoutParams.height = maxBounds[1];
-
-                videoView.setId(INLINE_VIDEO_PANE_ID);
-                return videoView;
+                videoView.setVisibility(VISIBLE);
             }
         } catch (InvalidReferenceException ire) {
             Log.e(TAG, "invalid video reference exception");
             ire.printStackTrace();
-            return getMissingMediaView(inlineVideoURI, "Invalid reference: " + ire.getReferenceString(), false);
+            showMissingMediaView(inlineVideoURI, "Invalid reference: " + ire.getReferenceString(), false, null);
```

Comment:
@shubham1g5 Do we need to localize this text?

Reply:
Also the information here isn't really informative imo. Would it be better if we say: `Can't show media due to: `?